build/cni/Makefile

```diff
@@ -22,13 +22,14 @@ CNI_RELEASE?=07a8a28637e97b22eb8dfe710eeae1344f69d16e
 CNI_TARBALL=cni-$(ARCH)-$(CNI_RELEASE).tar.gz
 CUR_DIR=$(shell pwd)
 OUTPUT_DIR=$(CUR_DIR)/output
+GOLANG_VERSION=1.7
```

Comment:
might want to pin this to an explicit version? otherwise `docker run` may not fetch a newer patch release.